### PR TITLE
Adopt random state from baycomp

### PR DESCRIPTION
**Pull request by Daniel300000**
Adopt the parameter “random_state” of baycomp for reproducible results.

As baycomp on PyPI is not up to date, use the most recent version from GitHub instead.

This could also be combined with my other pull request. "random_state" could also be passed to SignedRankTest().

### CODE REVIEW

File: autorank/autorank.py [hunk 1]
```diff
@@ -111,6 +111,9 @@ def autorank(data, alpha=0.05, verbose=False, order='descending', approach='freq
         automatically get the t-test/repeated measures ANOVA. With 'nonparametric' you automatically get Wilcoxon's
         signed rank test/Friedman test.
 
+    random_state (integer, default=None):
+        Seed for random state. Can be used to compute reproducible results.
```
Comment (sherbold) [inline autorank/autorank.py:115]: Please modify the second sentence to "Forwarded to Bayesian signed rank test to enable reproducible sampling and, thereby, reproducible results."

**Review by sherbold — COMMENTED** [review]
Small suggestion for the comment. Otherwise we just need to wait to see if there will be a new baycomp version before acceptance. Thanks!

**Comment by sherbold** [comment]
baycomp is now available in version 1.0.3 on Pypi. If you have time, please resolve the conflicts in this PR and use that version. than I can test and merge.

**Comment by Daniel300000** [comment]
Thank you for solving the problem with baycomp! I have resolved the conflicts in this pull request. Please let me know if there are any remaining problems.

**Comment by sherbold** [comment]
Thanks for the changes! Looks good 👍

**Pull request merged**